### PR TITLE
Updates to appium.io

### PR DESCRIPTION
**Pull request by rkavalap**
Some minor fixes to appium.io website.

### CODE REVIEW

**Comment by imurchie** [comment]
Where is this coming from?

File: index.html [hunk 1]
```diff
@@ -85,16 +76,13 @@ <h1 class="page-header" id="introducing-appium" data-localize="introducing-appiu
...
-                        src="http://www.youtube.com/embed/1J0aXDbjiUE"
-                        frameborder="0" allowfullscreen></iframe>
-                <iframe width="480" height="360"
-                        src="http://www.youtube.com/embed/oTIEt11cXNk"
-                        frameborder="0" allowfullscreen></iframe>
+                        src="https://www.youtube.com/embed/D5MF3ZLds0Q"
+                        frameborder="0" allowfullscreen></iframe>                
                 <iframe width="640" height="360"
-                        src="http://www.youtube.com/embed/ILKi6B54_sw"
+                        src="https://www.youtube.com/embed/WIuaT6nV9C4"
```
Comment (jlipps) [inline index.html]: I like the addition of these new videos, but not sure that we should get rid of the old ones

File: _data/slate.yml [hunk 2]
```diff
@@ -1,102 +1,4 @@
 en:
   - master
-  - 1.4
```
Comment (jlipps) [inline _data/slate.yml:3]: what's the reason for not building docs that aren't 'master'?

File: index.html (hunk 1 above)
Comment (rkavalap) [reply index.html]: @jlipps I am not a big fan of videos at the first place. But there is no harm in keeping them. But I didn't want to have something that was old but rather more recent.

File: _data/slate.yml (hunk 2 above)
Comment (rkavalap) [reply _data/slate.yml:3]: IMO I don't think end users have enough patience to even consume the existing 'master' documentation leave alone version'ed documentation. With agile approach and lot more frequent releases it does not make sense to maintain so many versions. We are bound to miss doc updates as well.

**Review by jlipps — APPROVED** [review]

**Pull request merged**